Products/DataCollector/plugins/zenoss/snmp/InterfaceMap.py

```diff
@@ -104,7 +107,10 @@ def process(self, device, results, log):
 
         # Add in IPv6 info
         ipv6table = tabledata.get("ipAddressIfIndex")
-        if ipv6table:
+        ipv6netmasktable = tabledata.get("ipAddressPrefix")
+        if ipv6table and ipv6netmasktable:
+            for key in ipv6table.keys():
+                ipv6table[key].update(ipv6netmasktable[key])
```

Comment:
This presumes that data for both IPv6 addresses _and_ netmasks must be available from the device if we want to model an IPv6 address.  What are the implications if we still used the retrieved IPv6 addresses regardless whether the netmasks could be retrieved?